apps/fhir/bluebutton/views/generic.py

```diff
@@ -1,6 +1,8 @@
-import logging
 import voluptuous
 import waffle
+import logging
+
+import apps.logging.request_logger as bb2logging
```

Comment:
@JFU-GIT Same here with an LGTM alert:

NEWModule 'apps.logging.request_logger' is imported with both 'import' and 'import from'